.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
     - chmod 777 ~/onFailure.sh
     - chmod 777 ~/onSuccess.sh
 install:
-    - mvn dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+    - mvn clean install dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
http://docs.travis-ci.com/user/build-configuration/#Build-Lifecycle

In this `install` is documented as preparing dependencies.  Adding `-N` would not recurse, but then what's the point?  A fact here is that builds will fail with the way this is setup right now.  Is it really a necessary step?